velox/substrait/SubstraitToVeloxExpr.cpp

```diff
@@ -73,21 +95,38 @@ SubstraitVeloxExprConverter::toVeloxExpr(
     case ::substrait::Expression_Literal::LiteralTypeCase::kBoolean:
       return std::make_shared<core::ConstantTypedExpr>(
           variant(substraitLit.boolean()));
+    case ::substrait::Expression_Literal::LiteralTypeCase::kI8:
```

Comment:
Nice. Would you add tests for this new logic? It would be nice to move this to a separate PR.

Reply:
> Nice. Would you add tests for this new logic? It would be nice to move this to a separate PR.

I add UT test about this type  here.  https://github.com/facebookincubator/velox/pull/2451/files#diff-89bf82c64f962bb1cfbb64db587ead2d783b512aec33e186d77abda98b073a66R293

Reply:
I'm not sure I understand how that tests is covering this code path. I'd assume this code path requires a query with literal / constants of various types. The unit test only uses columns. Also, the test uses different integers and floats, but not strings. Yet, this change adds branches for LiteralTypeCase::kString and LiteralTypeCase::kVarChar. Am I missing something?

Reply:
Let's move this logic into a separate PR and add a test. This would make reviewing easier and quicker.